spec/controllers/boards_controller_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "rails_helper"
+
+RSpec.describe BoardsController, type: :controller do
+
+  before do
+    ApplicationController.any_instance.stub(:current_user).and_return(true)
+  end
+
+  describe "GET index" do
+    it "assigns @boards" do
+      board = Board.create
+      get :index
+      expect(assigns(:boards)).to eq([board])
```

Comment:
wystarczy sprawdzic czy widok sie wyrenderował
expect(response).to render_tamplate :index